.github/workflows/haskell.yml

```diff
@@ -33,7 +33,7 @@ jobs:
     strategy:
       matrix:
         cabal: ["3.8.1.0"]
-        ghc: ["8.6.5", "8.8.4", "8.10.4", "9.0.2", "9.2.2", "9.4.5", "9.6.2"]
+        ghc: ["8.6.5", "8.8.4", "8.10.4", "9.0.2", "9.2.2", "9.4.5", "9.6.2", "9.8.1"]
```

Comment:
CI is failing due to this issue: https://github.com/haskellari/postgresql-libpq/issues/55

We may be able to fix this by adding an `allow-newer` flag to a `cabal.project`:

```
allow-newer:
  postgresql-libpq:Cabal
```

Reply:
Nope, this was the *Cabal* version that needed to change 😅